runtime/include/chpl-comm.h

```diff
@@ -109,13 +109,14 @@ void chpl_comm_wait_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles);
 int chpl_comm_try_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles);
 
 // Returns whether or not the passed wide address is known to be in
-// a communicable memory region - that is, a region for which it is
-// guaranteed that puts/gets will succeed without access violation or
-// other memory protection error.
+// a communicable memory region and known to be readable. That is,
+// GET to that address should succeed without an access violation
+// or other memory protection error.
 // Returns 1 if the entire passed region is known to be in the
 // registered memory region, or 0 if some or all of it is not (or
-// the communicable memory region is totally unknown).
-int chpl_comm_is_in_segment(c_nodeid_t node, void* start, size_t len);
+// the registered memory region is totally unknown as with e.g.
+// GASNET_SEGMENT=everything).
+int chpl_comm_addr_gettable(c_nodeid_t node, void* start, size_t len);
```

Comment:
Hmm, okay. After all this now I think maybe gettable is the right name after all, although you might adjust the function comment to say that for SEG=everything the reason it returns false/0 is because in that case being in the segment (which everything is, thus the name) does not imply being readable, unlike the others.